granulate_utils/linux/ns.py

```diff
@@ -180,7 +181,7 @@ def _find_inner_pid() -> Optional[int]:
         return None
 
     # We're searching `/proc`, so we only need to set our mount namespace
-    inner_pid = run_in_ns(["mnt"], _find_inner_pid, process.pid)
+    inner_pid = run_in_ns_wrapper(["mnt"], _find_inner_pid, process.pid)
```

Comment:
Like I commented on `granulate_utils/linux/proc_events.py`, this is not needed either.